config/src/comments.rs

```diff
@@ -274,13 +274,19 @@ fn comments() -> HashMap<String, String> {
 #how long a banned peer should stay banned
 #ban_window = 10800
 
-#maximum number of peers
-#peer_max_count = 125
+#maximum number of inbound peer connections
+#peer_max_inbound_count = 117
```

Comment:
We should just leave this as 125 or maybe even 128 for a nice round number.
It was arbitrary to begin with, definitely don't need to tweak them to add up to 125 in total.

Reply:
Increased the limit to 128 here.